src/screens/LandingPage/index.tsx

```diff
@@ -0,0 +1,110 @@
+import { Typography, Container, Stack, Divider } from "@mui/material";
+import { CustomButton } from "components";
+import { Footer, NavigationBar, LandingPageSection,LandingPageCard } from "../../components";
+import {
+    landingpageimage1,
+    landingpageimage2,
+    landingpageimage3,
+    landingimageCaseStudy,
+} from "assets";
+
+const CaseStudyStatments = {
```

Comment:
make a custom file for it in the mockups folder, and export it inside the index.ts in the mockups folder